src/test/java/org/folio/consortia/controller/UserTenantControllerTest.java

```diff
@@ -86,26 +89,17 @@ void shouldGetUserTenantByAssociationId() {
   @Test
   void shouldGetUserTenantList() throws Exception {
     var headers = defaultHeaders();
-    UUID consortiumId = UUID.fromString("7698e46-c3e3-11ed-afa1-0242ac120002");
+    UUID consortiumId = UUID.fromString(CONSORTIUM_ID);
     when(consortiumRepository.existsById(consortiumId)).thenReturn(true);
-    this.mockMvc.perform(get("/consortia/7698e46-c3e3-11ed-afa1-0242ac120002/user-tenants?limit=2&offset=1").headers(headers))
-      .andExpectAll(
-        status().isOk(),
-        content().contentType(MediaType.APPLICATION_JSON_VALUE));
+    this.mockMvc.perform(get("/consortia/7698e46-c3e3-11ed-afa1-0242ac120002/user-tenants?limit=2&offset=1").headers(headers)).andExpectAll(status().isOk(), content().contentType(MediaType.APPLICATION_JSON_VALUE));
   }
 
   @Test
-  void return404UserTenantNotFoundByAssociationId() throws Exception {
+  void shouldRemoveUserTenant() throws Exception {
```

Comment:
Probably there should be some another test name. It seems to be a test for successful removal base on the name, but waits for the error. Should be updated to understand which case is tested

Reply:
Thanks. I will fix it

Reply:
it would be good to see the reason why deletion should fail in the test name.

Reply:
I will rewrite, thanks